src/StepDefinition/DrupalFormParagraphTrait.php

```diff
@@ -68,17 +69,15 @@ protected function getParagraphFormFieldWrapper() {
    * @Then I add a paragraph :paragraph_type_label at slot number :slot
    */
   public function iAddAParagraphAtSlotNumber($paragraph_type_label, $slot) {
-    $xpath_add_button = "//div[contains(@class, 'field--widget-paragraphs')]//table[contains(@class, 'field-multiple-table--paragraphs')]/tbody/tr[contains(@class, 'paragraphs-features__add-in-between__row')][$slot]";
-    $xpath_paragraph_list = "//div[contains(@class, 'ui-dialog')]//ul";
-    $this->getSession()->getDriver()->click($xpath_add_button);
-    $this->getSession()->getDriver()
-      ->find($xpath_paragraph_list)[0]
-      ->findButton($paragraph_type_label)
-      ->click();
-
     $index = $slot - 1;
-    $condition = "return jQuery(\"div[data-drupal-selector=edit-field-paragraphs-$index]\").length";
-    $result = $this->getSession()->wait(5000, $condition);
+
+    $this->getSession()->executeScript("jQuery(jQuery(\".paragraphs-features__add-in-between__button\").get($index)).trigger('click');");
```

Comment:
this code seems to rely on pargraphs-features to be there. I guess we need to separate steps, or least clearly document that this step requires paragraphs-features to work.

Reply:
hmm it was relying on the same class before also:
`$xpath_add_button = "//div[contains(@class, 'field--widget-paragraphs')]//table[contains(@class, 'field-multiple-table--paragraphs')]/tbody/tr[contains(@class, 'paragraphs-features__add-in-between__row')][$slot]";`

Reply:
I see - that sucks. #merged then.